lua/easychat/modules/client/local_ui.lua

```diff
@@ -17,9 +17,18 @@ function panel:Think()
 end
 
 panel:SetWide(150)
+panel:SetZPos(-32768)

```

Comment:
Why this particular Z index?

Reply:
Minimal value

Reply:
I dont get why you need this?

Reply:
To make panel's hitbox not overlap with anything

Reply:
This is an unnecessarily high value. This will suffice.
```suggestion
panel:SetZPos(-5)

```

Reply:
My point is that why is there a need to change the z index of this panel in particular, what is it overlapping that it should be underneath everything?

Reply:
My guess they encountered a bug it overlaps with another UI layer but I find it highly implausible to happen in normal use.

Reply:
The whole point of this PR was to make message receivers panel to be rendered together with the chatbox on the same level, but the message receivers itself should not prevent other panels from being clickable + the panel has nothing to click so there's no point in it preventing any other panels from being clicked.